Add reminder about ignoring *.backup files via .npmignore (#15)
roydukkey/clean-package: README.md

}
```

> Edit your `.npmignore` file (or add one) and be sure to ignore the backup file. For example:

Reviewer: ```suggestion
When the "prepack" script executes, a backup of the original `package.json` will be created. Ensure this file doesn't make it into your release package.

One way to accomplish this is to add the following to your `.npmignore` file:

```

> ```text

Reviewer: ```suggestion
```ignore
```

> # Build Artifacts

Reviewer: ```suggestion
```
